nuxeo-core/nuxeo-core/src/main/java/org/nuxeo/ecm/core/migrator/AbstractRepositoryMigrator.java

```diff
@@ -84,8 +95,17 @@ protected void migrateRepository(String repositoryName) {
 
     protected abstract String probeSession(CoreSession session);
 
+    /**
+     * @deprecated since 11.1  Use {@link #migrateSession(MigrationContext, String, CoreSession)} instead
+     */
+    @Deprecated(since = "11.1", forRemoval = true)
     protected abstract void migrateSession(CoreSession session);
```

Comment:
We should provide a default implementation here. This is weird to have to implement something deprecated.

Reply:
yep, you right 👍 , we will implement it by calling the other method with `null` parameters